### DISCUSSION TITLE
Elasticsearch is getting reset

### DISCUSSION THREAD

**Discussion by snehal0904** [Sunbird-RC/community#233] [Issues]
I have set up the sunbird-rc setup. I am facing issues with the elastic search service. Elasticsearch service is getting reset, data is there in Postgres but getting clear from elastic search. I checked logs but did not get the reason for the auto-reset. Can someone advice, how I can debug this.
Also, is any sync tool available for Postgres and elastic search service?

@coolbung @ananya-jain1 @RinkleDang

**Comment by parthlawate** [comment]
@dileepbapat @tejash-jl could you help ?

**Comment by tejash-jl** [comment]
@snehal0904 are you running the services through docker-compose?
Can you share some more info on how you identified the data getting reset in ES?  Also, share the steps on how you are starting the services.

I think since ES in docker-compose doesn't have a persistent volume attached, the data will be lost on restart.

**Reply by snehal0904** [reply]
Yes, running service using docker-compose.
Identified ES is getting reset because data is present in the Postgres and indexes from the elastic search are blank. If we add new data, it is getting displayed.

After adding the new schema we are using - docker-compose up -d --force-recreate rg

I will try increasing the volume for the elastic search container and check this.

But I think we will need some sync script to add indexes for the existing data. Please share your inputs on this.

Thank you!!
